src/commonMain/kotlin/wizard/Entity.kt

```diff
@@ -30,22 +30,34 @@ val ProjectInfo.hasBrowser get() = platforms.any { it == ComposePlatform.Browser
 val ProjectInfo.packagePath get() = packageId.replace(".", "/")
 val ProjectInfo.safeName get() = name.replace(" ", "-")
 
+val ProjectInfo.kotlinVersionRef get() = "kotlin"
+val ProjectInfo.agpVersionRef get() = "agp"
+val ProjectInfo.composeVersionRef get() = "compose"
+
 data class Dependency(
     val title: String,
     val description: String,
     val url: String,
     val group: String,
     val id: String,
     val version: String,
-    val platforms: Set<ComposePlatform>
+    val versionCatalog: VersionCatalog,
+    val applyToModule: Boolean,
+    val platforms: Set<ComposePlatform>,
+)
+
+data class VersionCatalog(
+    val versionRefName: String,
+    val libraryName: String,
+    val libraryAccessor: String,
```

Comment:
I don't think we need the libraryAccessor property. There is a known rule hot gradle generates it: `libraryName.replace("-", ".").`
Let's make an extension for that.